### PR TITLE
feat: select example through url

### PR DESCRIPTION
**Pull request by wangqianwen0418**
select example through url for displaying examples in the static website

### CODE REVIEW

File: src/editor/editor.tsx [hunk 1]
```diff
@@ -135,13 +135,14 @@ function Editor(props: any) {
     const urlParams = qs.parse(props.location.search, { ignoreQueryPrefix: true });
     const urlSpec = urlParams?.spec ? JSONUncrush(urlParams.spec as string) : null;
     const urlGist = urlParams?.gist ?? null;
+    const urlExampleIndex = urlParams?.example ? examples.map(d => d.id).indexOf(urlParams.example as string) : null;
```
Comment (sehilyi) [inline src/editor/editor.tsx]: Currently, when a wrong example id is used, the site becomes entirely empty. How about changing this part to show a default (e.g., first) example if `example` does not exist?

**Review by sehilyi — CHANGES_REQUESTED** [review]
Thank you for this update, this looks good! I just left one comment about preventing from showing an empty page when a given `id` does not exist.

Also, just curious how `id`s will be managed in the `gosling-website`. I guess we will be having a JSON array that stores example ids and other related information (e.g., thumbnail), and we just need to update the array to add/remove certain examples to show in the `gosling-website`?

**Comment by wangqianwen0418** [comment]
Thanks! I have updated the code

**Comment by wangqianwen0418** [comment]
> I guess we will be having a JSON array that stores example ids and other related information (e.g., thumbnail), and we just need to update the array to add/remove certain examples to show in the gosling-website?

Yes, you are correct. The examples in gosling-website are currently documented in assets/example-list

**Review by sehilyi — APPROVED** [review]
Tested locally, and works well! I will merge this.

**Pull request merged**